.github/workflows/release-pypi.yml

```diff
@@ -1,20 +1,36 @@
 ---
-name: Release to PyPI
+name: Publish to PyPI
 
+# manually trigger this workflow
 on:
-  release:
-    types: [created]
-
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'The Git tag to create the release for'
```

Comment:
```suggestion
        description: 'The Git tag or branch to create or test a release for. Official releases should always be made from an existing tag.'
```

Reply:
This is not exactly true. I'm using action `softprops/action-gh-release` and the `tag` input will be passed to this action to the `tag_name` param. If the specified tag does not exist, it will create the tag from the current revision. I wanted to keep it simple but we can also have an additional input `ref` that (if set) will be the ref (i.e. SHA/branch/etc) to be used to create a new tag (if the tag does not yet exist).

Reply:
I just dropped the `or branch` for now.